docs/content/kubernetes-primer.md

```diff
@@ -187,7 +187,7 @@ Often you don't want to invent new variable names for iteration.  OPA provides t
 
 ### Builtins
 
-On line 5 the *builtin* `startswith` checks if one string is a prefix of the other.  The builtin `sprintf` on line 6 formats a string with arguments.  OPA has 50+ builtins detailed at [openpolicyagent.org/docs/policy-reference](../policy-reference).
+On line 5 the *builtin* `startswith` checks if one string is a prefix of the other.  The builtin `sprintf` on line 6 formats a string with arguments.  OPA has 150+ builtins detailed at [openpolicyagent.org/docs/policy-reference](../policy-reference).
```

Comment:
That link is wonky. Let's do
```suggestion
On line 5 the *builtin* `startswith` checks if one string is a prefix of the other. 
The builtin `sprintf` on line 6 formats a string with arguments.
OPA has 150+ builtins detailed at [the Policy Reference](../policy-reference/).
```

Reply:
we could also link straight to `../policy-reference/#built-in-functions`

Reply:
Fixed!